cdap-runtime-ext-remote-hadoop/src/main/resources/remote-hadoop.json

```diff
@@ -41,6 +41,20 @@
           }
         }
       ]
+    },
+    {
+      "label": "Advanced",
+      "properties": [
+        {
+          "widget-type": "textarea",
+          "label": "Initialization Action (optional)",
```

Comment:
We usually put the fact that it's optional in the description rather than in the label.